### ISSUE TITLE
Correct getDataBindings getters in README

### ISSUE THREAD

**Issue by appsforartists** [optimizely/nuclear-js#31]
In your examples, most of the KeyPaths are only a single level deep.  Since you encourage breaking the data architecture into modules, I presume that this is often the case in heavier apps too.

Allowing the user to specify KeyPaths as Strings (while using Arrays for deeply-nested paths) would make the code a bit less verbose.  It also would be consistent with the example for `getDataBindings`:

    getDataBindings() {
      return {
        // can reference a reactor KeyPath
        items: 'items',
        taxPercent: 'taxPercent',

I clearly haven't tried Nuclear yet.  If Strings are acceptable KeyPaths, the [documentation](https://github.com/optimizely/nuclear-js/#keypaths) should be updated to reflect that.  If not, should they be?  In any case, the KeyPaths used in `getDataBindings` should follow the same rules as the ones used in getters.

**Comment by mindjuice** [comment]
In previous releases, keypaths could, indeed, be strings.  That feature has been removed after moving to v1.0.0 though.  See **More Structured KeyPaths** in the [v1.0.0 Release Notes](https://github.com/optimizely/nuclear-js/releases).

**Comment by appsforartists** [comment]
That makes it a little more verbose.  Oh well.

In any case, the `getDataBindings` reference in the README should reflect that.

**Comment by jordangarcia** [comment]
Yes you are totally right, this needs to be fixed in the README.  Any PRs to fix minor things like these will be gladly merged.

Also the reasoning for this change was to reduce ambiguity for Getters that composed other Getters

```js
[
  ['foo', 'bar'],
  [
    ['baz', 'bah],
    (bah) => 2*bah
  ],
  (bar, bah) => bar + bah
],
```  

vs

```js
[
  ['foo', 'bar'],
  [
    'baz.bah', 
    bah => 2*bah
  ],
  (bar, bah) => baz + bah
] 
```


I personally prefer a singular way of doing things, especially on something as fundamental as keypaths.

**Comment by jordangarcia** [comment]
Also if you follow the module pattern talked about the in the README you are never using keypaths directly in Components.

They should be wrapped in a getter (a single keypath is a valid getter) and then exposed via `Chat.getters.messages` or something similiar.

**Comment by bhamodi** [comment]
Closed in #65.
